### ISSUE TITLE
Item do not decay when sector sleep

### ISSUE THREAD

**Issue by Jhobean** [Sphereserver/Source-X#535]
Since I using X, I notice some "annomaly" about the decay of item. When I entering a dungeon and someone pass few time before me, I see corps of monster on the ground. Since I go near of them they dissapear.

There is the same situation on mine where I can see old ore ont he ground and dissapear when i'm near them.

I think it can be 2 problems
1- Item do not decay on sleep and it should
2- Adjacent sector do not awake and it's why you see the item decay when you enter that sector.

This BUG may be the same bug than this issue: https://github.com/Sphereserver/Source-X/issues/467

**Comment by alexdan2** [comment]
Yep i confirm this issue, also is problematic for memory on summon or some other system based on memory timing, because if sector go to sleep memory on summon in sleep sector not decay --> summo not removed --> follower not update --> ecc ecc

**Comment by cbnolok** [comment]
I might be wrong but i think that on 56 the timer (which, upon reaching 0, causes the decay) froze if the sector was sleeping. If that's the case, the issue here is adjacent sector not waking up. Is that still happening?
